### PR TITLE
Code Refactor and Typo fix

### PR DESCRIPTION
**Pull request by prashant-andani**
Code Refactor 
Typo Fix

### CODE REVIEW

**Comment by react-linter** [comment]
<details>
  <summary>Details of bundled changes.</summary>

  <p>Comparing: 9e6c99ca2e56ff6ccdb9d7f816cd706228139cba...1152e2fa116d126d2223d841b02f8aaafbba437e</p>


  
## react-dom
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-dom.development.js | -0.1% | -0.1% | 627.88 KB | 627.3 KB | 146.85 KB | 146.7 KB | UMD_DEV
react-dom.development.js | -0.1% | -0.1% | 624 KB | 623.43 KB | 145.67 KB | 145.52 KB | NODE_DEV
react-dom-server.browser.development.js | -0.0% | -0.0% | 99.27 KB | 99.26 KB | 26.23 KB | 26.23 KB | UMD_DEV
**react-dom-server.browser.production.min.js** | **-0.0%** | **0.0%** | **14.81 KB** | **14.81 KB** | **5.69 KB** | **5.69 KB** | **UMD_PROD**
react-dom-server.browser.development.js | -0.0% | -0.0% | 95.39 KB | 95.38 KB | 25.29 KB | 25.29 KB | NODE_DEV
react-dom-server.node.development.js | -0.0% | -0.0% | 97.32 KB | 97.3 KB | 25.83 KB | 25.82 KB | NODE_DEV
ReactDOM-dev.js | -0.1% | -0.1% | 630.02 KB | 629.45 KB | 143.94 KB | 143.76 KB | FB_WWW_DEV
ReactDOMServer-dev.js | -0.0% | -0.0% | 96.51 KB | 96.49 KB | 24.84 KB | 24.84 KB | FB_WWW_DEV

## react-art
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-art.development.js | -0.1% | -0.2% | 411.86 KB | 411.29 KB | 92.41 KB | 92.26 KB | UMD_DEV
react-art.development.js | -0.2% | -0.2% | 343.84 KB | 343.27 KB | 75.27 KB | 75.12 KB | NODE_DEV
ReactART-dev.js | -0.2% | -0.3% | 332.67 KB | 332.09 KB | 70.01 KB | 69.83 KB | FB_WWW_DEV

## react-test-renderer
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-test-renderer.development.js | -0.2% | -0.2% | 348.35 KB | 347.78 KB | 75.83 KB | 75.68 KB | UMD_DEV
react-test-renderer.development.js | -0.2% | -0.2% | 344.47 KB | 343.9 KB | 74.85 KB | 74.69 KB | NODE_DEV
ReactTestRenderer-dev.js | -0.2% | -0.2% | 347.69 KB | 347.12 KB | 73.58 KB | 73.4 KB | FB_WWW_DEV

## react-noop-renderer
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-noop-renderer.development.js | -2.8% | -2.7% | 18.07 KB | 17.56 KB | 4.72 KB | 4.6 KB | NODE_DEV
react-noop-renderer-persistent.development.js | -2.8% | -2.7% | 18.19 KB | 17.68 KB | 4.74 KB | 4.61 KB | NODE_DEV

## react-reconciler
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
react-reconciler.development.js | -0.2% | -0.2% | 333.16 KB | 332.59 KB | 71.39 KB | 71.24 KB | NODE_DEV
react-reconciler-persistent.development.js | -0.2% | -0.2% | 331.78 KB | 331.21 KB | 70.83 KB | 70.68 KB | NODE_DEV

## react-native-renderer
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
ReactNativeRenderer-dev.js | -0.1% | -0.2% | 466.86 KB | 466.29 KB | 102.79 KB | 102.61 KB | RN_FB_DEV
ReactNativeRenderer-dev.js | -0.1% | -0.2% | 466.57 KB | 466 KB | 102.73 KB | 102.55 KB | RN_OSS_DEV
ReactFabric-dev.js | -0.1% | -0.2% | 457.12 KB | 456.54 KB | 100.38 KB | 100.2 KB | RN_FB_DEV
ReactFabric-dev.js | -0.1% | -0.2% | 457.15 KB | 456.58 KB | 100.4 KB | 100.21 KB | RN_OSS_DEV

  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/react-dom/src/events/BeforeInputEventPlugin.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const canUseTextInputEvent =
   canUseDOM && 'TextEvent' in window && !documentMode;
 
 // In IE9+, we have access to composition events, but the data supplied
-// by the native compositionend event may be incorrect. Japanese ideographic
```
Comment (gaearon) [inline packages/react-dom/src/events/BeforeInputEventPlugin.js]: This is not a typo. It's the event name.

File: packages/react-noop-renderer/src/createReactNoop.js [hunk 2]
```diff
@@ -551,18 +551,7 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
...
-        //   log(
-        //     '  '.repeat(depth + 1) + 'IN PROGRESS: ' + fiber.pendingWorkPriority,
-        //   );
-        //   logFiber(childInProgress, depth + 1);
-        //   if (fiber.child) {
-        //     log('  '.repeat(depth + 1) + 'CURRENT');
-        //   }
-        // } else if (fiber.child && fiber.updateQueue) {
-        //   log('  '.repeat(depth + 1) + 'CHILDREN');
-        // }
```
Comment (gaearon) [inline packages/react-noop-renderer/src/createReactNoop.js]: This code is here for debugging.

File: packages/react-reconciler/src/ReactFiberBeginWork.js [hunk 3]
```diff
@@ -1035,25 +1033,6 @@ function updateContextConsumer(current, workInProgress, renderExpirationTime) {
...
-      if (child.lastEffect) {
-        if (returnFiber.lastEffect) {
-          returnFiber.lastEffect.nextEffect = child.firstEffect;
-        }
-        returnFiber.lastEffect = child.lastEffect;
-      }
-    } while (child = child.sibling);
-  }
-  */
-
```
Comment (gaearon) [inline packages/react-reconciler/src/ReactFiberBeginWork.js]: This code is here for future reference.

Comment (prashant-andani) [reply packages/react-reconciler/src/ReactFiberBeginWork.js]: Thought! Can we not get the code from Git History for future reference?

Comment (prashant-andani) [reply packages/react-reconciler/src/ReactFiberBeginWork.js]: @gaeron add back the commented code for now.

**Comment by gaearon** [comment]
Sorry, we generally try to avoid merging changes that are purely stylistic. Happy to take a typo fix separately but this just doesn't look like it's going to be very helpful.

**Pull request closed without merging**